src/main/resources/node.linux.json

```diff
@@ -4,27 +4,18 @@
             {
                 "browserName": "firefox",
                 "maxInstances": <MAX_SESSION_FIREFOX>,
-                "instanceId": "<INSTANCE_ID>",
                 "seleniumProtocol": "WebDriver"
             },
             {
                 "browserName": "chrome",
                 "maxInstances": <MAX_SESSION_CHROME>,
-                "instanceId": "<INSTANCE_ID>",
```

Comment:
You can leave these fields on the individual browser slots actually (have this working locally).